docs/manuals/core/entities/index.md

```diff
@@ -4,7 +4,7 @@ This documentation focuses on providing necessary information to use the Taipy C
 the capabilities related to scenario management. It is assumed that the reader already knows the [Taipy Core
 concepts](../concepts/index.md) described in a previous documentation.
 
-It is also assumed in the next sections that [`my_config.py`](../my_config.py) module contains a Taipy configuration
+It is also assumed in the next sections that [`my_config.py`](https://github.com/Avaiga/taipy-doc/blob/develop/docs/manuals/core/my_config.py) module contains a Taipy configuration
```

Comment:
We cannot have an absolute link for that. Here you are pointing to the develop version of the file which might change over time and over the versions.

Reply:
What we do in gui is we have a 'doc' directory at the top, with files in there.
There's no real choice but to point with an absolute url...

Reply:
`my_config.py` also requires `my_function.py`, and (probably) the `sales.csv` too.
So if we can generate a download link, it should download all 3 files, or at least 2 python files.

Reply:
Well @jrobinAV, I'm not sure the difference we can make between 'a link' and 'a download link'...

Reply:
We can create a zip file including the 2 python files and `sales.csv`, but I don't think that's a good idea.

Also, in `docs/manuals/core/entities/pipeline-mgt.md`, the content of `my_config.py` and `my_function.py` is also shown. So I think it's better if we have a dedicate page to show the example config, example data, and only a link reference to this page anywhere else.

Reply:
OK I created a dedicated ticket for that. Thx !

https://github.com/Avaiga/taipy-doc/issues/261